src/main/scala/com/rbmhtechnology/calliope/javadsl/TransactionalEventProducer.scala

```diff
@@ -114,9 +114,6 @@ object TransactionalEventProducer {
     def withProducerProperty(key: String, value: String): Settings[A] =
       copy(delegate.withProducerProperty(key, value))
 
-    def withProducerProvider(producerProvider: ProducerProvider[A]): Settings[A] =
```

Comment:
I guess this means we can also delete the type aliases for `ProducerProvider` in java- and scaladsl